clang/test/CodeGenSYCL/Inputs/builtin.cpp

```diff
@@ -0,0 +1,3 @@
+void a(__builtin_va_list);
+using ::a;
+
```

Comment:
I believe we can remove this file and move the content to `clang/test/CodeGenSYCL/debug-info-file-prefix-map.cpp` directly.